### ISSUE TITLE
How to call the Ocelot API Gateway (using dot net core) from angular 7

### ISSUE THREAD

**Issue by Ramakrish** [ThreeMammals/Ocelot#1029]
Hi Team 

We are implemented the Ocelot in dot net core project..

We are created the reroutes for two different kind of API's.

While running the  Ocelot with reroutes in my browsers it provide the expected result.

We need to call the same Ocelot API gateway reroute URL from angular 7 project.

Is it possible to call Ocelot APIGateway URL from angular 7??
Kindly help me on that..

**Comment by Loongle** [comment]
you need configuration your stratup. additional cors configuration

**Comment by wast** [comment]
Yes you can. Read [documentation ](https://ocelot.readthedocs.io/en/latest/introduction/gettingstarted.html).

Close this. It's not an issue. Use Stack Overflow if you can't manage to do it.

**Comment by raman-m** [comment]
Hi @Ramakrish! 
Hare Krishna! Hare Rama! :smile: Forever!

> Is it possible to call Ocelot API gateway URL from angular 7??

Angular is front-end framework. Any front-end framework requires to deploy special type of BFF gateway which has special configuration and setup.
In general Ocelot is good choice to deploy BFF gateway instance: we support main protocols: `http` + `https`, `ws` + `wss`.
We have some problems with OPTIONS and CORS policies, see #799. But we believe, experienced ASP.NET developer can solve these problems.
